api/tests/unit/infrastructure/jobs/cpf-export/handlers/send-email_test.js

```diff
@@ -0,0 +1,53 @@
+const { expect, sinon } = require('../../../../../test-helper');
+const sendEmail = require('../../../../../../lib/infrastructure/jobs/cpf-export/handlers/send-email');
+const { cpf } = require('../../../../../../lib/config');
+const moment = require('moment-timezone');
+
+describe('Unit | Infrastructure | jobs | cpf-export | send-email', function () {
+  let cpfExternalStorage;
+  let mailService;
+  beforeEach(function () {
+    cpfExternalStorage = {
+      getPreSignUrlsOfFilesModifiedAfter: sinon.stub(),
+    };
+    mailService = {
+      sendCpfEmail: sinon.stub(),
+    };
+  });
+
+  it('should get files from external storage modified after the previous planner cron', async function () {
+    // given
+    sinon.stub(cpf, 'plannerJob').value({ cron: '0 0 15 01 *' });
+
+    // when
+    await sendEmail({ cpfExternalStorage, mailService });
+
+    // then
+    const expectedDate = moment.tz('01-15', 'MM-DD', 'Europe/Paris').toDate();
```

Comment:
Pour être un peu plus explicite sur le cron: 

```suggestion
    const day = '15';
    month = '01'
    sinon.stub(cpf, 'plannerJob').value({ cron: `0 0 ${day} ${month} *` });

    // when
    await sendEmail({ cpfExternalStorage, mailService });

    // then
    const expectedDate = moment.tz(`${month}-${day}`, 'MM-DD', 'Europe/Paris').toDate();
```

Reply:
C'est plus clair en effet